src/plugins/setup.ts

```diff
@@ -126,10 +126,13 @@ export default createPlugin(
 									message:
 										"Waiting for authentication response from the browser...",
 								});
-								const { authToken } = await minDelay(
+								const { authToken, canceled } = await minDelay(
 									requestAuthentication(context, cancellationToken),
 								);
-								if (cancellationToken.isCancellationRequested) {
+								if (canceled) {
```

Comment:
can we rename this to `cancelled` everywhere for consistency?
```suggestion
								if (cancelled) {
```